faucet/valve.py

```diff
@@ -612,6 +616,15 @@ def _update_stack_link_state(self, ports, now, other_valves):
                     ofmsgs_by_valve[valve].extend(valve.flood_manager.add_vlan(vlan))
                 for port in valve.dp.stack_ports:
                     ofmsgs_by_valve[valve].extend(valve.host_manager.del_port(port))
+
+                if valve.flood_manager.graph and not graph_updated:
+                    with open('/etc/faucet/dp_graph.json', 'w') as outfile:
```

Comment:
We should ditch this as soon as we can, so we don't inadvertently rely on it!

Reply:
Will remove it once the daq changes are ready.